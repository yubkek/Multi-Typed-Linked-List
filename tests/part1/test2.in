NEW 0
VIEW -1.0
VIEW 1.0
TYPE HELLO
NEW 0
