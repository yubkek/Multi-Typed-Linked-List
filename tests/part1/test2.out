List 0:
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW
INVALID COMMAND: TYPE
List 1: